### ISSUE TITLE
Staking `force_unbond`

### ISSUE THREAD

**Issue by kianenigma** [paritytech/polkadot-sdk#414]
Currently, as a nominator or validator, your request to fully unbond can fail if you are not chilled yet. 

The right fix is for the UI to detect this and batch it with a `chill`.

A `force_unbond` can help clarify this, and is not that much overhead on the chain runtime. under the hood, it would be an optional chill, followed by unbond.

**Comment by kianenigma** [comment]
> Just to clarify, can this `force_unbond` be a permissionless call if the staker is already chilled?
> 
> The ability to remove inactive stakers from storage like this would also allow us to target those who have a unique stash & controller pair, & could speed up controller account removal.

Not really, this is a different concern. You might want to chill yourself while staying bonded for various reasons, and I don't see why we should prevent users from being in this state. 

I personally know stakers who generally keep their funds bonded but chilled for safety.

**Comment by dharjeezy** [comment]
Hello @kianenigma i will like to work on this.

**Comment by Ank4n** [comment]
> Currently, as a nominator or validator, your request to fully unbond can fail if you are not chilled yet.
> 
> The right fix is for the UI to detect this and batch it with a `chill`.
> 
> A `force_unbond` can help clarify this, and is not that much overhead on the chain runtime. under the hood, it would be an optional chill, followed by unbond.

A better solution might be to improve the checks in unbond which chills if it is full unbond. But it should never allow ledger active balance to be between 0 and MinNominatorBond.
